### PR TITLE
fishPlugins.colored-man-pages: init at unstable-2022-04-30

### PR DESCRIPTION
**Pull request by jocelynthode**
###### Description of changes

This fish plugins colorizes man pages to make them more readable.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/shells/fish/plugins/colored-man-pages.nix
Comment (pacien) [inline pkgs/shells/fish/plugins/colored-man-pages.nix:10]: Doesn't build:

```
~/src/nixpkgs add-color-man-pages ≡
❯ nix build --impure --expr 'with import ./. {}; wrapFish {
     pluginPkgs = [ pkgs.fishPlugins.colored-man-pages ];
   }'
error: builder for '/nix/store/wc2pzlcp5hv9b31zwrzfzhkg3l8srxvg-source.drv' failed with exit code 1;
       last 8 log lines:
       >
       > trying https://github.com/patrickf1/colored_man_pages.fish/archive/unstable-2022-04-30.tar.gz
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       >   0    14    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       > curl: (22) The requested URL returned error: 404
       > error: cannot download source from any mirror
       For full logs, run 'nix log /nix/store/wc2pzlcp5hv9b31zwrzfzhkg3l8srxvg-source.drv'.
error: 1 dependencies of derivation '/nix/store/68rr008m1slx4245qzll2219jmx7b766-fishplugin-colored-man-pages-unstable-2022-04-30.drv' failed to build
error: 1 dependencies of derivation '/nix/store/8g389r5sng65s2clx74hjk3js2kxg8hs-fish.drv' failed to build
```

Since there's no tagged release, we might need to use the commit hash matching the date:

```suggestion
    rev = "f885c2507128b70d6c41b043070a8f399988bc7a";
```

**Review by pacien — CHANGES_REQUESTED** [review]

File: pkgs/shells/fish/plugins/colored-man-pages.nix
Comment (jocelynthode) [reply pkgs/shells/fish/plugins/colored-man-pages.nix]: Ah sorry I did a last minute modification to match the version name as in other fish plugins and forgot to change it there. I did not notice it on my end as the package path was already in my nix store and Nix did not try to re download it.

It's now fixed thanks!

**Review by pacien — APPROVED** [review]

**Comment by jocelynthode** [comment]
After your approval should I do something specific or does it get merged automatically?

**Comment by pacien** [comment]
Quoting Jocelyn Thode (2022-11-17 19:55:04)
> After your approval should I do something specific or does it get
> merged automatically?

The proposed changes usually get merged quickly once reviewed and
approved, if nobody else has any other remark. This has to be done
manually by someone who has write access to the repository.

Nothing left to do on your side beside waiting for now.

**Pull request merged**
